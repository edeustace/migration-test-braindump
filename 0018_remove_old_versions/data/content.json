[
   {
      "_id":{
         "$oid":"51da9b2223064367505f8891"
      },
      "version":{
         "current":true
      },
      "taskInfo" : {
        "title" : "one"
      }
   },
   {
      "_id":{
         "$oid":"51da9b2223064367505f8892"
      },
      "version":{
         "current":true
      },
      "taskInfo" : {
        "title" : "two"
      }
   },
   {
      "_id":{
         "$oid":"51da9b2223064367505f8893"
      },
      "version":{
         "current":false
      },
      "taskInfo" : {
        "title" : "three"
      }
   }
 ]
