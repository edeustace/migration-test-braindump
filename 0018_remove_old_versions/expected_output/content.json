[
   {
      "_id":{"_id" : {"$oid":"51da9b2223064367505f8891"}, "version" : 0 },
      "taskInfo" : {
        "title" : "one"
      }
   },
   {
      "_id":{"_id" : {"$oid":"51da9b2223064367505f8892"}, "version" : 0 },
      "taskInfo" : {
        "title" : "two"
      }
   }
]
